src/components/Book.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+export default function Book(props) {
+  const { topic, title, author } = props;
+  return (
+    <div className="container">
+      <ul>
+        <li>{topic}</li>
+        <li>{title}</li>
+        <li>{author}</li>
+      </ul>
+      <button type="button">UPDATE PROGRESS</button>
+    </div>
```

Comment:
- [ ] Kindly this `div` should contains a remove button also as required in the requirement of the project `Should display the list of books (empty at this point but it should be ready for the data) with the Remove button (no funcionality yet). `
- [ ] As required the style or any functionality for now is not require. Add the remove button to your project and let's see it approve :handshake: 

Your design
![Screenshot from 2022-06-03 03-48-02](https://user-images.githubusercontent.com/77842237/171763400-52980126-7afa-4a65-973e-1935c21f9a55.png)

Project with remove b
![Screenshot from 2022-06-03 03-43-14](https://user-images.githubusercontent.com/77842237/171763440-343ddbc0-f1ef-42ad-b09e-8a4c652f2704.png)
utton